packages/core/data/observable/index.ts

```diff
@@ -116,7 +116,7 @@ export class Observable implements ObservableDefinition {
 			throw new TypeError('callback must be function.');
 		}
 
-		const events = eventNames.trim().split(/,\s*/);
+		const events = eventNames.trim().split(/\s*,\s*/);
```

Comment:
```suggestion
		const events = eventNames.trim().split(EVENT_SEPARATOR_RE);
```
Given this regex is repeating a few times, perhaps we could extract it to a const like 

```ts
const EVENT_SEPARATOR_RE = /\s*,\s*/
```

I believe that would also be slightly faster (don't have benchmarks to support my claims tho)